Contact.html

```diff
@@ -0,0 +1,178 @@
+                            finsweet@gmail.com
+                        </p>
+                        <p class="card__icons card__icons--title-link">
+                            Help Email support
+                        </p>
+                    </div>
+                </div>
+                <div class="icon">
+                    <img src="images/message.png" alt="" class="icon__image">
+                    <div class="card__icons card__icons--title">
+                        <p class="card__icons card__icons--title-contact">
+                            sales@finsweet.com
+                        </p>
+                        <p class="card__icons card__icons--title-link">
+                            Sales Enquiry
+                        </p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+
+
+    <footer class="footer">
+        <div class="footer__background">
+      
+        </div>
+        <p class="title title--middle">
+          Are you ready to grow your <br/> business with us?
+        </p>
+        <p class="footer__subtitle">
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy.
+        </p>
+        <button class="footer__button">
+          Free trial
+        </button>
+        <div class="footer-down">
+          <div class="footer-down__item footer-down__item--middle">
+            <div class="footer-down__list">
+                <p class="footer-down__title">
+                    Company Info
+                </p>
+                <ul class="footer-down__links">
+                    <li class="footer-down__link">About Us</li>
+                    <li class="footer-down__link">Carrier</li>
+                    <li class="footer-down__link">We are hiring</li>
+                    <li class="footer-down__link">Blog</li>
+                </ul>
+            </div>
+            <div class="footer-down__list">
+                <p class="footer-down__title">
+                    Company Info
+                </p>
+                <ul class="footer-down__links">
+                    <li class="footer-down__link">About Us</li>
+                    <li class="footer-down__link">Carrier</li>
+                    <li class="footer-down__link">We are hiring</li>
+                    <li class="footer-down__link">Blog</li>
+                </ul>
+            </div>
+            <div class="footer-down__list">
+                <p class="footer-down__title">
+                    Company Info
+                </p>
+                <ul class="footer-down__links">
+                    <li class="footer-down__link">About Us</li>
+                    <li class="footer-down__link">Carrier</li>
+                    <li class="footer-down__link">We are hiring</li>
+                    <li class="footer-down__link">Blog</li>
+                </ul>
+            </div>
+            <div class="footer-down__list">
```

Comment:
`footer-down` is a block inside another block `footer`. it's not valid. no blocks inside blocks in BEM